### PR TITLE
Some fixes and editorial checklist update

### LINKED ISSUE #47: Headings instructions in the style guide is duplicated
In the Text section, there are instructions for "Headings" and "Titles", both includes information about capitalisation, in Titles mentions acronyms. 

https://www.infectious-diseases-toolkit.org/contribute/style_guide

### LINKED ISSUE #65: Editors checklist
Add to the editor's checklist - what an editor should do when a page is updated? (check RDMKit for duplicit information, etc...)

### LINKED ISSUE #67: Issue on page: News
I would like to report an issue on the News page at `/about/news`. Description of the issue:

Clicking on the "propose changes..." pencil icon leads to a 404 page on GitHub with the following message:
`The 'elixir-europe/infectious-diseases-toolkit' repository doesn't contain the '_data/news.yaml' path in 'main'.`

### PR DESCRIPTION
**Pull request by bedroesb**
- news edit me path (closes #67 )
- events edit me path
- code of conduct in sidebar
- editorial checklist RDMkit check (closes #65 )
- Specify about which title we talk in the style sheet, aligning with RDMkit (https://github.com/elixir-europe/rdmkit/pull/983) (closes #47)

### CODE REVIEW

File: contribute/editors_checklist.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ summary: Checklist for editors before approving and merging a pull request (PR).
    * `coordinators`(only used in national pages + they must be listed as `contributors` as well)
    * `resources`
 5. Items in the "[Tools and resources spreadsheet](https://docs.google.com/spreadsheets/d/13tqfSbgivokfEkxGXPRFShVhCmO4VuTQRe4uQgJOMbc)" are tagged with already existing (merged) `page_id` from "Pathogen characterisation, Social and economic impact, Human biomolecular data, Human clinical and health data, Showcase" and that Bert has been informed of the changes.
-6. The content is within the scope of the Infectious Diseases Toolkit scope, [style](/contribute/style_guide) and conforms to the appropriate templates.
+6. The content is within the scope of the Infectious Diseases Toolkit (prevent overlap with the [RDMkit](https://rdmkit.elixir-europe.org/) and link towards it instead), conforms to the appropriate templates and the [style](/contribute/style_guide) is matching our standards.
```
Comment (rabuono) [inline contribute/editors_checklist.md:21]:
```suggestion
6. The content is within the scope of the Infectious Diseases Toolkit (prevent overlap with the [RDMkit](https://rdmkit.elixir-europe.org/) and link towards it instead), conforms to the appropriate templates and the [style](/contribute/style_guide).
```
Is it sufficient to stop at "style"?

**Review by rabuono — COMMENTED** [review]
Looks good to me!
Just a minor comment on the text of the checklist.

**Comment by bedroesb** [comment]
better indeed

**Comment by bedroesb** [comment]
if you like it, approve it and merge 😄

**Review by rabuono — APPROVED** [review]

**Pull request merged**